src/Components/Aspire.Keycloak/README.md

```diff
@@ -0,0 +1,105 @@
+# Aspire.Keycloak library
+
+Adds JwtBearer and OpenId Connect authentication to the project via a [Keycloak](https://www.keycloak.org).
+
+## Getting started
+
+### Prerequisites
+
+- A Keycloak server instance
+- A Keycloak realm
+- For JwtBearer authentication, a configured audience in the Keycloak realm
+- For OpenId Connect authentication, the ID of a client configured in the Keycloak realm
+
+### Install the package
+
+Install the .NET Aspire Keycloak library with [NuGet](https://www.nuget.org):
+
+```dotnetcli
+dotnet add package Aspire.Keycloak
+```
+
+## Jwt bearer authentication usage example
+
+In the _Program.cs_ file of your ASP.NET Core API project, call the `AddKeycloakJwtBearer` extension method to add JwtBearer authentication, using a connection name, realm and any required JWT Bearer options:
+
+```csharp
+builder.Services.AddAuthentication()
+                .AddKeycloakJwtBearer("keycloak", realm: "WeatherShop", options =>
```

Comment:
Can we use 4 spaces indent in this file?

Reply:
According to VS Code, it uses 4 spaces indent. Am I missing something?